portal/src/main/webapp/js/src/study-view/data/StudyViewProxy.js

```diff
@@ -221,11 +221,11 @@ var StudyViewProxy = (function() {
                         var _parts = _lists[i].split('\t');
```

Comment:
@zheins @jjgao : this part could at least use some inline comments explaining why we skip when _parts.length < 5, why we can expect an empty space at the end of the line and what are parts [0] to [4]....could you please add it so that future troubleshooting is easier?